hvcc/generators/c2daisy/c2daisy.py

```diff
@@ -44,33 +45,42 @@ def compile(clazz, c_src_dir, out_dir, externs,
             source_dir = os.path.join(out_dir, "source")
             shutil.copytree(c_src_dir, source_dir)
 
-            # supply a custom board description file, if it exists
             if daisy_meta.get('board_file', {}):
```

Comment:
this should not have the `, {}` otherwise it'll never return None

Reply:
While it's true that an empty dictionary doesn't evaluate to `None`, it will evaluate to false in an if statement. However, the board_file isn't expected to be a dictionary at all, so I'm not sure why that's there.